chapter-10-workload-identities/README.md

```diff
@@ -40,17 +40,24 @@ See the [PKI setup](./base/pki/README.md) documentation for details.
 ./2-deploy-pki.sh
 ```
 
-### 3. Deploy the Service Mesh and API gateway
+### 3. Deploy the Service Mesh
 
-Next, deploy the Istio service mesh, which provides its own API gateway.\
-This gateway can initiate mTLS connections when routing to upstream components.\
+Next, deploy the Istio service mesh, to enable transparent mTLS inside the cluster.\
 See the [Service mesh setup](./base/service-mesh/README.md) for details on the Istio SPIRE integration.
 
 ```bash
 ./3-deploy-service-mesh.sh
 ```
 
```

Comment:
There is an extra step now, and the existing API gateway deployment is used.
This avoids duplication and provides continuity - all of the gateway patterns continue to work when Istio is used.